File: C++/ArrayDemo.cpp
#include <iostream>
using namespace std;

int main() {
    int maxSize;

    cout << "enter size of array: " << endl;
    cin >> maxSize;

    int arr[maxSize];

    cout << "enter array elements: " << endl;
    for (int i = 0; i < maxSize; i++) {
        cin >> arr[i];
    }

    cout << "your entered array: " << endl;
    for (int a = 0; a < maxSize; a++) {
        cout << arr[a] << "  ";
    }

    return 0;
}

File: Python/ArrayDemo.py
def main():
    max_size = int(input("Enter size of array: "))
    arr = []

    print("Enter array elements:")
    for _ in range(max_size):
        arr.append(int(input()))

    print("Your entered array:")
    print("  ".join(str(x) for x in arr))

if __name__ == "__main__":
    main()